src/main/java/se/kth/sda/skeleton/commentlikes/CommentLikeRepository.java

```diff
@@ -0,0 +1,12 @@
+package se.kth.sda.skeleton.commentlikes;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * Represents the database of domain type Comment as a JPA Repository
```

Comment:
`Represents the database of domain type **CommentLike** as a JPA Repository`

Reply:
Done